src/dbg/expressionfunctions.h

```diff
@@ -5,20 +5,26 @@
 class ExpressionFunctions
 {
 public:
-    using CBEXPRESSIONFUNCTION = std::function<duint(int argc, const duint* argv)>;
+    using CBEXPRESSIONFUNCTION             = std::function<duint(int argc, const duint* argv)>;
+    using CBEXPRESSIONFUNCTIONWITHUSERDATA = std::function<duint(int argc, const duint* argv, void* user)>;
 
     static void Init();
     static bool Register(const String & name, int argc, CBEXPRESSIONFUNCTION cbFunction);
+    static bool Register(const String & name, int argc, CBEXPRESSIONFUNCTIONWITHUSERDATA cbFunction, void* user);
     static bool Unregister(const String & name);
     static bool Call(const String & name, const std::vector<duint> & argv, duint & result);
     static bool GetArgc(const String & name, int & argc);
 
 private:
     struct Function
     {
+        CBEXPRESSIONFUNCTION cbFunction;

```

Comment:
`std::function` also has captures and since the userdata has to be provided during registration there don't have to be changes to this particular definition.